radio/src/opentx.cpp

```diff
@@ -1879,6 +1888,10 @@ void opentxInit()
     // no backlight mode off on color lcd radios
     g_eeGeneral.backlightMode = e_backlight_mode_keys;
   }
+  if (g_eeGeneral.backlightBright <= BACKLIGHT_LEVEL_MIN)
+    g_eeGeneral.backlightBright = BACKLIGHT_LEVEL_MIN+5;
```

Comment:
What is actually the minimum here? Is that `BACKLIGHT_LEVEL_MIN`, `BACKLIGHT_LEVEL_MIN + 1`, or `BACKLIGHT_LEVEL_MIN + 5`?

We could just as well set `BACKLIGHT_LEVEL_MIN` to the value corresponding to the current `BACKLIGHT_LEVEL_MIN + 5` and just enforce relative to `BACKLIGHT_LEVEL_MIN`.

Reply:
> What is actually the minimum here? Is that `BACKLIGHT_LEVEL_MIN`, `BACKLIGHT_LEVEL_MIN + 1`, or `BACKLIGHT_LEVEL_MIN + 5`?

The idea here is to allow the user to set the backlight to `BACKLIGHT_LEVEL_MIN + 1` on purpose, but when something goes wrong and it is set to off, you get, as a failsave, a level where you can read something on the display when you are outside.



> We could just as well set `BACKLIGHT_LEVEL_MIN` to the value corresponding to the current `BACKLIGHT_LEVEL_MIN + 5` and just enforce relative to `BACKLIGHT_LEVEL_MIN`.

This does not work, as `BACKLIGHT_LEVEL_MIN` is also the lower bound fot the backlight off brightness, or do you mean we always add 5 to the on brightness and limit it to `BACKLIGHT_LEVEL_MAX`?

Reply:
What I did not take into account is the inverted baclight level usage on color lcd radios. I pushed an update that fixes this and also removed to offsets. You are right about the offsets, it is better to just increase the minimum value, if needed.

thx